.github/workflows/ci.yml

```diff
@@ -48,6 +48,10 @@ jobs:
       - name: Run unit tests
         run: yarn test:ci
 
+      - name: Run vitest unit tests
+        if: matrix.node == 18
```

Comment:
I think once we deprecate Node 16 and have Node 20, we can just exclude the non supported versions. Ty for adding theses changes in our CI pipeline!